lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -130,10 +137,29 @@ def test_no_tab_flag_unset(self, enable_proctored_exams, enable_timed_exams):
         self.assertNotIn(self.proctoring_link, response.content)
         self.assertNotIn('Allowance Section', response.content)
 
+    def test_review_dashboard(self):
+        """
+        The exam review dashboard will appear for backends that support the feature
+        """
+        self.setup_course(True, True)
+        backend = TestBackendProvider()
+        config = apps.get_app_config('edx_proctoring')
+        with patch.object(config, 'backends', {'test': backend}):
+            create_exam(
+                course_id=self.course.id,
+                content_id='test_content',
+                exam_name='Final Test Exam',
+                time_limit_mins=10,
+                backend='test',
+            )
+            response = self._assert_proctoring_tab_is_available()
+            self.assertIn('Review Dashboard', response.content)
+
     def _assert_proctoring_tab_is_available(self):
         """
         Asserts that proctoring tab is available for logged in user.
         """
         response = self.client.get(self.url)
         self.assertIn(self.proctoring_link, response.content)
         self.assertIn('Allowance Section', response.content)
+        return response
```

Comment:
Nit: I think it's clearer to me if `_assert_proctoring_tab_is_available` only performs assertions and doesn't return anything (the response in this case, which I'd return from another function).